src/client/components/simulator/index.js

```diff
@@ -35,6 +42,23 @@ export default function Simulator() {
     }
   }, [controller, currentPos]);
 
+  useEffect(() => {
+    if (controller) {
+      if (isPlaying) {
+        console.log("Play");
+        controller.pixiApp.startTime = performance.now();
+        controller.pixiApp.waveSuferTime = timeData.time;
+        controller.pixiApp.state = { controlRecord, posRecord };
```

Comment:
What is a `pixiApp.state` means here? Why you store data in `controller.pixiApp` but not in the `controller `?
Besides, you can have a `set` function in the controller and just pass arguments into it, which will makes the code here more simple.